src/main/java/car/Car.java

```diff
@@ -0,0 +1,51 @@
+package car;
+
+import java.util.Random;
+
+public class Car {
+    private final String name;
```

Comment:
```suggestion
    private final CarName name;
```
`"자동차 이름은 5자 미만만 가능 합니다."` 의 조건을 지켜야하는 `객체`를 생성해보면 어떨까요?